hs_geo_raster_resource/tests/test_raster_metadata_user_zone.py

```diff
@@ -136,3 +136,31 @@ def test_metadata_in_user_zone(self):
 
         # resource core metadata is deleted after resource deletion
         self.assertEqual(CoreMetaData.objects.all().count(), 0)
+
+        # test adding file from user zone to existing empty resource in hydroshare zone
+        # even there is no file uploaded to resource initially, there are default extended
+        # automatically metadata created
+        _, _, metadata, _ = utils.resource_pre_create_actions(
```

Comment:
I don't think this function call is anymore needed as the raster resource type doesn't handle pre resource create signal.